app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
+  before_action :find_post
+
+  def create
+    # @user = User.find(params[:author_id])
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.author_id = @user.id
+
+    if @comment.save
+      redirect_to user_post_path(@user, @post), notice: 'Successful!'
+    else
+      flash.now[:notice] = 'Comment could not be created'
```

Comment:
- [ ] Also here. I would recommend adding a `new` method to create an instance.

```ruby
class CommentsController < ApplicationController

    ...

    def new
      @comment = Comment.new
    end
end
```

Reply:
Thanks, @Dipeshtwis for your kind suggestion and feedback!. This suggestion is implemented on commit 54ec112.